src/style.css

```diff
@@ -0,0 +1,150 @@
+.page {
+  font-family: Roboto, Helvetica, sans-serif;
+}
+
+.content {
+  --big_text_color: #060b35;
+  --small_text_color: #616070;
+
+  margin: 0;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+.card {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+
+  width: 200px;
+  height: 408px;
```

Comment:
proofread the checklist
![image](https://user-images.githubusercontent.com/98764641/210343410-61ef9f42-02b9-4948-bc7d-e6e2ded41971.png)